src/pspm_dcm_inv.m

```diff
@@ -64,7 +64,6 @@
 %   │                 [numeric, default: 0.1, unit: second]
 %   │                 minimum dispersion (standard deviation) for flexible
 %   │                 responses.
-%   ├──────.missing:  data points to be disregarded by inversion.
```

Comment:
Please revert, change to "missing_data", and list under "model".

Reply:
`.missing_data` is now added at line 37 and used in line 548.